packages/react/src/slider/thumb/useSliderThumb.ts

```diff
@@ -110,17 +109,16 @@ export function useSliderThumb(parameters: useSliderThumb.Parameters): useSlider
     return {
       position: 'absolute',
       [{
-        horizontal: 'left',
-        'horizontal-reverse': 'right',
+        horizontal: 'insetInlineStart',
```

Comment:
@vladmoroz Since we decided that with DirectionProvider, and users are still responsible for setting HTML `dir` themselves, I wanted to use a logical property here instead of left/right, what do you think?
(The new DirectionProvider doc should mention this as well)

Reply:
This makes sense, but I'm still seeing `dir="rtl"` on the slider in the direction provider demo. Are you planning to remove it? (I don't mind it)

https://deploy-preview-1017--base-ui.netlify.app/react/utilities/direction-provider

<img width="700" alt="image" src="https://github.com/user-attachments/assets/2193b663-aca7-4447-a313-dab024632414">

Reply:
Yup, I've updated the (new) demo as well: https://github.com/mui/base-ui/pull/1017/commits/569bdf0a065d07a4d2914bbc0cd04d8443d98edd
@vladmoroz If its ok I'll merge it later

Reply:
Sure. That'd be a separate PR?

Reply:
Nope! updated here in this commit: https://github.com/mui/base-ui/commit/569bdf0a065d07a4d2914bbc0cd04d8443d98edd @vladmoroz